eng/packages/http-client-csharp/generator/Azure.Generator/src/Visitors/LroVisitor.cs

```diff
@@ -88,7 +88,7 @@ private static void UpdateMethodSignature(ScmMethodProvider method)
             var returnType = (responseType, method.IsProtocolMethod) switch
             {
                 (null, _) => typeof(Operation),
-                (not null, true) => new CSharpType(typeof(Operation), typeof(BinaryData)),
+                (not null, true) => new CSharpType(typeof(Operation<>), typeof(BinaryData)),
```

Comment:
Hmm, was this causing issues? There is a spector scenario for it - https://github.com/Azure/azure-sdk-for-net/blob/main/eng/packages/http-client-csharp/generator/TestProjects/Spector/http/azure/core/lro/standard/src/Generated/StandardClient.cs#L25

Reply:
This only throw on debug, got it while debugging with mgmt

Reply:
https://github.com/microsoft/typespec/blob/b06b74a7ab21d82b1dd9d7e85ef3ea4684d6aea5/packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/src/Primitives/CSharpType.cs#L121-L132